package/harvester-os/Dockerfile

```diff
@@ -10,4 +10,4 @@ RUN rm -f /system/oem/05_network.yaml
 
 ARG HARVESTER_INSTALLER_VERSION
 
-RUN echo GRUB_ENTRY_NAME="\"Harvester $HARVESTER_INSTALLER_VERSION\"" >> /etc/os-release
+RUN echo GRUB_ENTRY_NAME="\"HarvesterOS $HARVESTER_INSTALLER_VERSION\"" >> /etc/os-release
```

Comment:
`HarvesterOS` actually does not exist, we can leave it as Harvester thanks.

Reply:
Removed the term.